src/index.css

```diff
@@ -147,7 +147,7 @@
 }
 
 .fc .fc-timegrid-slots .fc-timegrid-slot {
-    height: 100px;
+    height: 70px;
```

Comment:
This compromises visibility of locations on 1hr slots. 
![image](https://user-images.githubusercontent.com/59380148/149756201-760eaadd-7dc1-4bf1-96f7-e3d19ab06039.png)
Granted, this happens anyways at 100px now, but it wasn't an issue before with the fluid container on larger displays. Possibly could be worth changing this to a min-height max-height sort of situation, or adding an overflow? Not sure really. @pl4nty any ideas?

Reply:
I tend to agree with you, but I'm not much good at styling. Maybe @OliverBalfour has some ideas from our discussion in #82? that was where the 100px/80px were originally implemented

Reply:
Yeah 100px empirically seemed like the best tradeoff between things being cut off on 1h slots and enough stuff fitting on one screenfull. I'd like to keep it at 100px for desktop

Reply:
Perhaps we keep it at 100px and add a screenshot export button if a lot of screens don't fit the full range of events vertically at a time. I've just reported this as #92

Reply:
I can see why you'd want to keep it at 100px. The tradeoff is that while nothing is cut-off at 100px, the usefulness as a daily planner is reduced.

If it is preferred to leave at 100px, I'll just change it for my own screenshtos haha

Reply:
> Perhaps we keep it at 100px and add a screenshot export button if a lot of screens don't fit the full range of events vertically at a time. I've just reported this as #92

whoops, had my tab open for a long time and didn't see this before my last comment

that sounds like the best solution